clients/admin-ui/src/features/system/SystemFormTabs.tsx

```diff
@@ -37,45 +38,20 @@ const ToastMessage = ({
 }: {
   onViewDatamap: () => void;
   onAddPrivacyDeclaration: () => void;
-}) => {
-  const linkButtonProps = {
-    variant: "link",
-    textDecor: "underline",
-    textColor: "gray.700",
-    fontWeight: "medium",
-    // allow lines to wrap
-    display: "initial",
-    cursor: "pointer",
-  };
-  return (
-    <Box>
-      <Text fontWeight="700">System has been saved successfully</Text>
-      <Text textColor="gray.700" whiteSpace="inherit">
-        Your system has been added to your data map. You can{" "}
-        <Button
```

Comment:
this is technically unrelated to this PR, but this PR also needed a button link in a toast, which we did here. so I extracted what we did here out into its own component `ToastLink` and used it here, too.